Documentation/grpc-metadata.md

```diff
@@ -223,3 +223,49 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
     stream.SetTrailer(trailer)
 }
 ```
+
+## Updating metadata in the interceptor - server side
+
+Server side metadata updating in the interceptor examples are available [here](../examples/features/metadata_interceptor/server/main.go).
+
+#### Unary interceptor
+
+To read and update metadata in the interceptor under unary call, the server needs to retrieve it from RPC context and update it through this RPC context.
+
+```go
+func SomeInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+	// get the metadata from context
```

Comment:
Nit: Maybe replace with:

```golang
// Get the incoming metadata from the RPC context, and add a new 
// key-value pair to it.
md, ok := metadata.FromIncomingContext(ctx)
md.Append("key1", "value1")

// Create a context with the new metadata and pass it to handler.
ctx = metadata.NewIncomingContext(ctx, md)
return handler(ctx, req)

```

Reply:
Got it, sorry for my bad statement in the comment